integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/mock/ImageCatalogBasicTest.java

```diff
@@ -0,0 +1,63 @@
+package com.sequenceiq.it.cloudbreak.testcase.mock;
+
+import static com.sequenceiq.it.cloudbreak.dto.mock.CheckCount.atLeast;
+import static com.sequenceiq.it.cloudbreak.mock.ImageCatalogMockServerSetup.responseFromJsonFile;
+
+import javax.inject.Inject;
+
+import org.testng.annotations.Test;
+
+import com.sequenceiq.it.TestParameter;
+import com.sequenceiq.it.cloudbreak.action.v4.credential.CredentialCreateAction;
+import com.sequenceiq.it.cloudbreak.action.v4.imagecatalog.ImageCatalogCreateRetryAction;
+import com.sequenceiq.it.cloudbreak.client.ImageCatalogTestClient;
+import com.sequenceiq.it.cloudbreak.context.Description;
+import com.sequenceiq.it.cloudbreak.context.TestContext;
+import com.sequenceiq.it.cloudbreak.dto.credential.CredentialTestDto;
+import com.sequenceiq.it.cloudbreak.dto.imagecatalog.ImageCatalogTestDto;
+import com.sequenceiq.it.cloudbreak.dto.mock.HttpMock;
+import com.sequenceiq.it.cloudbreak.dto.mock.endpoint.ImageCatalogEndpoint;
+import com.sequenceiq.it.cloudbreak.mock.ImageCatalogMockServerSetup;
+import com.sequenceiq.it.cloudbreak.testcase.AbstractMinimalTest;
+
+public class ImageCatalogBasicTest extends AbstractMinimalTest {
+
+    @Inject
+    ImageCatalogMockServerSetup imageCatalogMockServerSetup;
+
+    @Inject
+    private ImageCatalogTestClient imageCatalogTestClient;
+
+    @Inject
+    private TestParameter testParameter;
+
+    @Test(dataProvider = TEST_CONTEXT)
+    @Description(
+            given = "image catalog valid URL",
+            when = "calling create image catalog with that URL",
+            then = "getting image catalog response so the creation success")
+    public void testIC(TestContext testContext) {
+        String imgCatalogName = resourcePropertyProvider().getName();
+
+        testContext
+                .as()
+                .given(HttpMock.class)
+                .whenRequested(ImageCatalogEndpoint.Base.class).getCatalog().thenReturn(
+                (model, uriParameters) -> imageCatalogMockServerSetup
+                        .patchCbVersion(responseFromJsonFile("imagecatalog/catalog.json"), testParameter))
+                .whenRequested(ImageCatalogEndpoint.Base.class).head()
+                .thenReturnHeader("Content-Length", "38")
+                .thenReturn((model, uriParameters) -> "")
```

Comment:
is it mean: `Should be returned with empty?`

Reply:
yes, i'll clarify this